### PR TITLE
Multiple test adapter paths issue

### PR DESCRIPTION
**Pull request by hvinett**
## Description
Notify IDE that multiple test adapters are dropped by picking up one adapter with the highest verson.

## Related issue
#2121

### CODE REVIEW

File: src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs [hunk 1]
```diff
@@ -311,6 +317,12 @@ public void SendTestMessage(TestMessageLevel level, string message)
             this.communicationManager.SendMessage(MessageType.TestMessage, payload);
         }
 
+        public void TestRunMessageHandler(object sender, TestRunMessageEventArgs e)
+        {
+            var payload = new TestMessagePayload { MessageLevel = e.Level, Message = e.Message };
```
Comment (mayankbansal018) [inline src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs]: Check for errors, & warnings, Only those should be sent to IDE

File: src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs [hunk 2]
```diff
@@ -311,6 +317,15 @@ public void SendTestMessage(TestMessageLevel level, string message)
             this.communicationManager.SendMessage(MessageType.TestMessage, payload);
         }
 
+        public void TestRunMessageHandler(object sender, TestRunMessageEventArgs e)
+        {
+            if(e.Level!=TestMessageLevel.Informational)
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs]: make this explicit, == warning || == error

File: src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs [hunk 3]
```diff
@@ -41,6 +43,8 @@ public class DesignModeClient : IDesignModeClient
 
         protected Action<Message> onAckMessageReceived;
 
+        private TestSessionMessageLogger testSessionMessageLogger;
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs]: Why do we need this as a private member ?

File: src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs [hunk 4]
```diff
@@ -311,6 +317,20 @@ public void SendTestMessage(TestMessageLevel level, string message)
...
         }
 
+        /// <summary>
+        /// Sends the test session logger warning and error messages to IDE; 
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        public void TestRunMessageHandler(object sender, TestRunMessageEventArgs e)
+        {
+            if(e.Level==TestMessageLevel.Error || e.Level ==TestMessageLevel.Warning)
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs]: Nit: why isn't the spacing correct here ?

**Review by singhsarab — APPROVED** [review]

**Pull request merged**